src/index.html

```diff
@@ -17,11 +21,401 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="header__logo-img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
```

Comment:
According to the checklist, the class name 'is-active' should be written as 'nav__link--active' to follow the BEM naming convention.